gke/kubeconfig

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
```

Comment:
sao lại để kubeconfig lên git nhỉ, phải gen config trong action hoặc store ở nơi an toàn.

Reply:
e đã xóa rồi ạ